### DISCUSSION TITLE
Enemy nameplate disappear while enemy cast spells

### DISCUSSION THREAD

**Discussion by xqmi** [Backupiseasy/ThreatPlates#361] [Q&A]
Hi, I've noticed this during few M+ runs.

When the targeted enemy start casting, their whole nameplate just disappear...

I notice it does that more often when the enemy cast uninterruptable spells. Happens on all types of enemies (boss,minibosss,trash mobs...) and happen on both interruptable and uninterruptable.

Anyone else experiencing this?

Im not sure where to post this problem, someone plz let me know.

**Comment by Backupiseasy** [comment]
Ah, sorry, xqmi - I don't seem to get notifications reliable when someone opens a discussion. Also, it's not shown in the tab list above that there is a new one and I never check this :) For such bugs, you could always just open a issue. 

Regarding your issue: Not sure, I do not experience that. If this does happen with every cast, can you check that the transparency for casting units (General - Transparency - Situational Transparency) is set to 0% (and not to, e.g. 100%). It's not like that by default, so it should not be that, but it never hurts to check.

**Comment by Backupiseasy** [comment]
Ah, also, to check if it is settings-related or not, just create a new profile with default settings and see if this still happens.
